packages/help-center/src/hooks/use-get-history-chats.ts

```diff
@@ -54,18 +52,17 @@ const getOdieConversationsWithNoSupportInteractions = (
 };
 
 /**
- * Checks if the message content is one of the predefined "no message" contents.
- * @param text The message content to check.
- * @returns True if the message content is one of the predefined "no message" contents, false otherwise.
+ * Checks whether the last message from the specified conversation is not empty nor a predefined '--' token.
  */
-const isNoMessageContent = ( text: unknown ) => {
+const isValidLastMessageContent = ( conversation: OdieConversation | ZendeskConversation ) => {
+	const { text } = getLastMessage( { conversation } ) || {};
+
 	if ( text === null || text === undefined ) {
-		return NO_MESSAGE_CONTENTS.includes( text );
+		return false;
 	}
-	if ( typeof text === 'string' ) {
-		return NO_MESSAGE_CONTENTS.includes( text.trim() );
-	}
-	return false;
+
+	// '--' is a token returned by AI models when conversations should be deflected to human agents
```

Comment:
Quite not accurate stating that it's returned by "AI models". It's actually our own Odie configuration file.

Reply:
Good point, I'll change that.